api/v1alpha1/edgedevice_types.go

```diff
@@ -44,6 +44,8 @@ type MetricsReceiverConfiguration struct {
 	RequestNumSamples int64  `json:"requestNumSamples,omitempty"`
 	TimeoutSeconds    int64  `json:"timeoutSeconds,omitempty"`
 	URL               string `json:"url,omitempty"`
+	// CA is a base64 encoding of CA certificate
+	CA string `json:"ca,omitempty"`
```

Comment:
Also, CA should be an array of strings, if it has an intermediate certificate need to have all the chain to get it working as expected.

Reply:
no need. the string can be a concatenation of CAs/certs. CA is saved to file and file can include multiple PEM certs/CAs.